src/Raven.Server/Documents/Indexes/Static/TimeSeries/TimeSeriesSegmentObjectInstance.cs

```diff
@@ -137,7 +129,7 @@ private static ObjectInstance CreateValue(Engine engine, DynamicTimeSeriesSegmen
             {
                 var value = new JsObject(engine);
 
-                value.FastSetDataProperty(nameof(entry.Tag), entry._entry.Tag?.ToString());
+                value.FastSetDataProperty(nameof(entry.Tag), entry._entry.Tag is not null ? new LazyJsString(entry._entry.Tag) : Null);
```

Comment:
I'm wondering if `Null` should not be changed to `DynamicJsNull.ExplicitNull` ?

Reply:
The old code relied in implicit conversion by Jint which hit this code path:

https://github.com/sebastienros/jint/blob/b75ca9f2fb5c6bec87b816bd3af8029504e4283a/Jint/Native/JsValue.cs#L284-L287

So changing to `ExplicitNull` would change behavior, not sure what the ramifications in RavenDB would be. Shall I change?

Reply:
I think it could be a small bug from previous versions, in general, during indexing, we are trying to return those DynamicJsNull values where possible to support null propagation. I do not think it should change the behavior here.

Reply:
I force-pushed a change which now uses `ExplicitNull` if value is `null`.